### PR TITLE
Beatmaps: redo page

### PR DESCRIPTION
**Pull request by MegaApplePi**
- get rid of questions section... why is it here?

---

(glossary)
- glossary: add mod requests term

---

(beatmaps/packs)
- add page

---

(redirects)
- add more keys
  - `ranked` -> `Beatmaps/#ranked` (updates this one)
  - `approved` -> `Beatmaps/#approved`
  - `qualified` -> `Beatmaps/#qualified`
  - `pending` -> `Beatmaps/#pending`
  - `mod_request` -> `Glossary/#mod-request`
  - `modders` -> `Glossary/#modder`
  - `modded` -> `Modding`

### CODE REVIEW

File: wiki/Beatmaps/en.md [hunk 1]
```diff
@@ -1,207 +1,86 @@
...
 
-Ranked and approved beatmaps both are beatmaps that two [BNs](/wiki/Beatmap_Nomination_Group) approved of. They can influence the stats on a player's profile, which is what makes them different from [pending](#pending) or [graveyarded](#graveyard) beatmaps.
+Both ranked and approved beatmaps requires approval from two  [Beatmap Nominators](/wiki/Beatmap_Nominators).
+They can influence the stats on a player's profile which is what makes them different from a [pending](#pending) or [graveyarded](#graveyard) beatmap.
 
-#### Ranked
+### Ranked
 
-Ranked Beatmaps have a heart ![Heart icon](/wiki/shared/Heart.gif) icon on the thread and on the in-game song selection.
+Ranked beatmaps use the NAMEOFICON icon (![NAMEOFICON icon](/wiki/shared/ranked.gif)) in the song select screen.
```
Comment (MegaApplePi) [inline wiki/Beatmaps/en.md]: nts: change `NAMEOFICON` to what the icon is called

File: wiki/Beatmaps/Packs/en.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+### Beatmap Pack Listing
+
+Download all the (-/Approved/Taiko/osu!mania) beatmaps packs from the **Standard** category and nowhere else.
+The (-/Approved/Taiko/osu!mania) beatmaps packs will contain all the songs in the other 3 categories.
+
+The beatmaps packs can be found and downloaded from: [osu! Beatmaps Packs page](https://osu.ppy.sh/p/packlist).
+
+### Torrenting
+
+*Notice: Torrenting osu!beatmaps seems to of had died off approximately 2 years ago and counting... (within the border of osu! and it's forums)*
```
Comment (peppy) [inline wiki/Beatmaps/Packs/en.md]: bad english "of had died off"

File: wiki/Beatmaps/Packs/en.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Beatmaps Packs
+
+_Main page: [Beatmaps](/wiki/Beatmaps)_
+
+Beatmaps packs, as the title suggests, are a `.zip` files containing beatmaps.
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: The title doesnt suggest that its exactly a zip file

File: wiki/Beatmaps/Packs/en.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+The name of the pack is based on what they contain (e.g. `Approved Beatmap Pack #7` would only contain [approved](#approved) beatmaps).
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: #approved? No header like this?

File: wiki/Beatmaps/Packs/en.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+It is worth noting that the most packs are:
+
+- Roughly either ~100MB (Taiko/osu!mania) or ~200MB (-/Approved)
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: osu!taiko

File: wiki/Beatmaps/Packs/en.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+- Always contain videos (if included)
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: if included or always?

File: wiki/Beatmaps/Packs/en.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+| Category | Beatmap types |
+| -------- | ------------- |
+| Standard | Ranked or Approved songs |
+| Chart | Fortnightly Chart challenge songs |
+| Theme | General theme of the songs |
+| Artist/Album | Based on Artist/Album |
+
+## Obtaining Every Beatmap
+
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: That header doesnt make any sense; no information about how to obtain every beatmap

File: wiki/Beatmaps/Packs/en.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+This is for those who really want to *fully complete your song list* (and not just download your favorite songs).
+
+Keep in mind that you will need quite a lot of digital space to storage these beatmaps.
+As beatmap packs (and torrents) also downloads with the video and other beatmap skinable elements.
+
+### Beatmap Pack Listing
+
+Download all the (-/Approved/Taiko/osu!mania) beatmaps packs from the **Standard** category and nowhere else.
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: osu!taiko

File: wiki/Beatmaps/Packs/en.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+The (-/Approved/Taiko/osu!mania) beatmaps packs will contain all the songs in the other 3 categories.
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: osu!taiko

File: wiki/Beatmaps/en.md [hunk 10]
```diff
@@ -1,207 +1,86 @@
 # Beatmaps
 
-## What are "Beatmaps"?
+Beatmaps, or _map_ for short, are game levels in [osu!](/wiki/osu!_glossary), like the stages in the [DS games](/wiki/Glossary/#ds-games).
```
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: or maps

File: wiki/Beatmaps/en.md [hunk 11]
```diff
@@ -1,207 +1,86 @@
...
+In their most basic form, beatmaps have a song various [hit objects](/wiki/hit_objects) placed according to the rhythm of the music.
+They can also have a background image, video, and/or [storyboard](/wiki/storyboards).
 
-*Beatmaps listing page: <https://osu.ppy.sh/p/beatmaplist>*
+Creators of beatmaps are called beatmappers or mappers for short.
+Beatmaps are uploaded for everyone to see and play with using the [BSS](/wiki/BSS), and are then [modded](/wiki/modded).
```
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: juat everyone to play i think

File: wiki/Beatmaps/en.md [hunk 12]
```diff
@@ -1,207 +1,86 @@
...
+Qualified beatmaps use the check icon (![Check icon](/wiki/shared/qualified.gif)) in the song select screen.
 
-#### Pending
+These beatmaps are the last stage of the beatmapping and modding process.
+Before the map start when playing, a dialoge will display, noting the status of the beatmap will cause:
 
-**Pending beatmaps** are beatmaps that are usually complete but are awaiting [rank](#ranked) / [approval](#approved) status. [Mappers](/wiki/Glossary/#beatmapper) must seek out [modding](/wiki/Modding) from the community and two [BATs](/wiki/Beatmap_Nomination_Group). Beatmaps will remain pending until they have been [ranked](#ranked), [approved](#approved), or [graveyarded](#graveyard), the latter automatically occurring after 4 weeks of inactivity on the corresponding beatmap thread. Pending beatmaps, like graveyarded beatmaps do not affect gameplay statistics.
+- No [pp](/wiki/pp) will be awarded.
+- All scores will be deleted when the status is changed.
```
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: All local scores?

File: wiki/Beatmaps/en.md [hunk 13]
```diff
@@ -1,207 +1,86 @@
...
+  - If the status is changed to ranked, approved, or back to pending; all scores will be wiped!
```
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: all local scores

File: wiki/Beatmaps/en.md (hunk 12 above)
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: misread, dont mind

File: wiki/Beatmaps/en.md (hunk 13 above)
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: misread

File: wiki/Beatmaps/Packs/en.md (hunk 6 above)
Comment (RaikireHiuduo) [inline wiki/Beatmaps/Packs/en.md]:
```Markdown
- Contain videos (if included)
```

File: wiki/Beatmaps/Packs/en.md [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+
+_Main page: [Beatmaps](/wiki/Beatmaps)_
+
+Beatmaps packs, as the title suggests, are a `.zip` files containing beatmaps.
+The name of the pack is based on what they contain (e.g. `Approved Beatmap Pack #7` would only contain [approved](#approved) beatmaps).
+It is worth noting that the most packs are:
+
+- Roughly either ~100MB (Taiko/osu!mania) or ~200MB (-/Approved)
+- Always contain videos (if included)
+- The beatmaps are either in the form of `.zip` or `.osz` file
```
Comment (RaikireHiuduo) [inline wiki/Beatmaps/Packs/en.md]: You can remove the `.zip` notice. It is for _ancient_-level beatmap packs which should be extinct by now.

File: wiki/Beatmaps/Packs/en.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+
+## Categories
+
+Beatmaps packs split into 4 categories:
```
Comment (RaikireHiuduo) [inline wiki/Beatmaps/Packs/en.md]:
```Markdown
Beatmaps packs are split into 4 categories:
```

File: wiki/Beatmaps/Packs/en.md [hunk 16]
```diff
@@ -0,0 +1,48 @@
+# Beatmaps Packs
```
Comment (RaikireHiuduo) [inline wiki/Beatmaps/Packs/en.md]: `Beatmap Packs` ?

File: wiki/Beatmaps/en.md [hunk 17]
```diff
@@ -1,207 +1,86 @@
 # Beatmaps
 
-## What are "Beatmaps"?
+Beatmaps, or _map_ for short, are game levels in [osu!](/wiki/osu!_glossary), like the stages in the [DS games](/wiki/Glossary/#ds-games).
+In their most basic form, beatmaps have a song various [hit objects](/wiki/hit_objects) placed according to the rhythm of the music.
```
Comment (RaikireHiuduo) [inline wiki/Beatmaps/en.md]: `In their most basic form, a beatmap have a song and various [hit objects](/wiki/hit_objects) placed according to the rhythm of the music.`

File: wiki/Beatmaps/en.md [hunk 18]
```diff
@@ -1,207 +1,86 @@
...
+They can also have a background image, video, and/or [storyboard](/wiki/storyboards).
 
-*Beatmaps listing page: <https://osu.ppy.sh/p/beatmaplist>*
+Creators of beatmaps are called beatmappers or mappers for short.
+Beatmaps are uploaded for everyone to see and play with using the [BSS](/wiki/BSS), and are then [modded](/wiki/modded).
+If and once a beatmap is deemed good enough, it is [ranked](#ranked) or, if fulfilling certain criteria, [approved](#approved).
```
Comment (RaikireHiuduo) [inline wiki/Beatmaps/en.md]: `Once a beatmap is deemed good enough, it is [ranked](#ranked) or if fulfilling certain criteria, [approved](#approved).`

File: wiki/Beatmaps/en.md (hunk 1 above)
Comment (RaikireHiuduo) [inline wiki/Beatmaps/en.md]: From @peppy  in Discord: `double upwards chevron`

File: wiki/Beatmaps/Packs/en.md (hunk 4 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: whoops, this was carried over from the Beatmaps page (the page that originally had this)

File: wiki/Beatmaps/Packs/en.md (hunk 5 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: isn't this a pack name?

Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: correction, this is the name of the pack :/

File: wiki/Beatmaps/Packs/en.md (hunk 7 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: what do you suggest then?

File: wiki/Beatmaps/Packs/en.md (hunk 8 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: once again, that's the pack name and not fully referring to the game mode itself

File: wiki/Beatmaps/Packs/en.md (hunk 9 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: ditto

File: wiki/Beatmaps/en.md (hunk 11 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/en.md]: what do you mean?

File: wiki/Beatmaps/Packs/en.md (hunk 16 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: lol... just `# Packs`

File: wiki/Beatmaps/Packs/en.md (hunk 14 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: will remove overall then... since `.osz` is the thing nowadays :p

File: wiki/Beatmaps/Packs/en.md (hunk 7 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: using `## Fully Completing Your Song List` instead

Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: or just delete it

File: wiki/Beatmaps/en.md (hunk 11 above)
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: `for everyone to see and play`
`for everyone to play`?

File: wiki/Beatmaps/en.md [hunk 19]
```diff
@@ -1,207 +1,86 @@
 # Beatmaps
 
-## What are "Beatmaps"?
+Beatmaps, or _maps_ for short, are game levels in [osu!](/wiki/osu!_glossary), like the stages in the [DS games](/wiki/Glossary/#ds-games).
```
Comment (peppy) [inline wiki/Beatmaps/en.md]: we never call them "maps", please stick with beatmaps

**Comment by peppy** [comment]
remove newlines

File: wiki/Beatmaps/en.md [hunk 20]
```diff
@@ -1,207 +1,86 @@
...
+Beatmaps are uploaded for everyone to play with using the [BSS](/wiki/BSS), and are then [modded](/wiki/modded).
+Once a beatmap is deemed good enough, it is [ranked](#ranked) or if fulfilling certain criteria, [approved](#approved).
 
-Also called map for short, they are game levels in [osu!](/wiki/Glossary/#osu), like the stages in the [DS games](/wiki/Glossary/#ds-games). In their most basic form, beatmaps have a song and [Hit Circles](/wiki/Hit_Objects), [Sliders](/wiki/Hit_Objects), and/or [Spinners](/wiki/Hit_Objects) placed according to the rhythm of the music, but they can also have a background image or video, or even a [Storyboard](/wiki/Storyboards). Creators of beatmaps are called beatmappers or mappers for short. Beatmaps are uploaded for everyone to see and play with using the [BSS](/wiki/Glossary/#bss), and are then [modded](/wiki/Modding). If/Once a beatmap is deemed good enough, it is [ranked](#ranked) or, if fulfilling certain criteria, [approved](#approved).
+You can find beatmaps in the [Beatmaps Listing](https://osu.ppy.sh/beatmapsets) page on osu!web.
 
-### What types of Beatmaps are there?
+## Beatmap Statuses
```
Comment (peppy) [inline wiki/Beatmaps/en.md]: Ranked Statuses

File: wiki/Beatmaps/en.md [hunk 21]
```diff
@@ -1,207 +1,86 @@
 # Beatmaps
 
-## What are "Beatmaps"?
+Beatmaps, or _maps_ for short, are game levels in [osu!](/wiki/osu!_glossary), like the stages in the [DS games](/wiki/Glossary/#ds-games).
+In their most basic form, a beatmap have a song and various [hit objects](/wiki/hit_objects) placed according to the rhythm of the music.
+They can also have a background image, video, and/or [storyboard](/wiki/storyboards).
 
-*Beatmaps listing page: <https://osu.ppy.sh/p/beatmaplist>*
+Creators of beatmaps are called beatmappers or mappers for short.
```
Comment (peppy) [inline wiki/Beatmaps/en.md]: we refer to them as "creator" everywhere, this isn't needed

File: wiki/Beatmaps/en.md [hunk 22]
```diff
@@ -1,207 +1,86 @@
...
+Beatmaps are uploaded for everyone to play with using the [BSS](/wiki/BSS), and are then [modded](/wiki/modded).
```
Comment (peppy) [inline wiki/Beatmaps/en.md]: "can be uploaded to the BSS"

Comment (peppy) [inline wiki/Beatmaps/en.md]: no comma before and (invalid grammar)

Comment (peppy) [inline wiki/Beatmaps/en.md]: "and are then run through an approval process called [modding](...) in order to maintain the high standards of playability

**Comment by TPGPL** [comment]
Could it contain the section about downloading beatmaps (from current Welcome)?

File: wiki/Beatmaps/Packs/en.md [hunk 23]
```diff
@@ -0,0 +1,48 @@
+# Packs
+
+_Main page: [Beatmaps](/wiki/Beatmaps)_
+
+A beatmaps pack is a `.zip` file containing beatmaps.
+The name of the pack is based on what they contain (e.g. `Approved Beatmap Pack #7` would only contain [approved](/wiki/approved) beatmaps).
+
+It is worth noting that the most packs are:
+
+- Roughly either ~100MB (Taiko/osu!mania) or ~200MB (-/Approved)
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: wut is dis (-/Approved) link

File: wiki/Beatmaps/Packs/en.md [hunk 24]
```diff
@@ -0,0 +1,48 @@
...
+Beatmaps packs are split into 4 categories:
+
+| Category | Beatmap types |
+| -------- | ------------- |
+| Standard | Ranked or Approved songs |
+| Chart | Fortnightly Chart challenge songs |
+| Theme | General theme of the songs |
+| Artist/Album | Based on Artist/Album |
+
+## Fully Completing Your Song List
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: Delete this section

File: wiki/Beatmaps/Packs/en.md [hunk 25]
```diff
@@ -0,0 +1,48 @@
...
+
+Keep in mind that you will need quite a lot of digital space to storage these beatmaps.
+As beatmap packs (and torrents) also downloads with the video and other beatmap skinable elements.
+
+### Beatmap Pack Listing
+
+Download all the (-/Approved/Taiko/osu!mania) beatmaps packs from the **Standard** category and nowhere else.
+The (-/Approved/Taiko/osu!mania) beatmaps packs will contain all the songs in the other 3 categories.
+
+The beatmaps packs can be found and downloaded from: [osu! Beatmaps Packs page](https://osu.ppy.sh/p/packlist).
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: Could use the page from new osu!web

File: wiki/Beatmaps/Packs/en.md [hunk 26]
```diff
@@ -0,0 +1,48 @@
...
+
+### Torrenting
+
+_Notice: Beatmap torrents seem to have died off within the osu!community._
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: Italic into ">" styling

File: wiki/Beatmaps/Packs/en.md [hunk 27]
```diff
@@ -0,0 +1,48 @@
...
+
+- [Beatmap-Torrents](https://osu.ppy.sh/forum/t/147478)
+  - [HiraOsu! - Torrents](http://osu.hiramiya.me/torrents.htm) by [Hiramiya](https://osu.ppy.sh/u/1313441)
+
+These torrents contain the 10, 100, 200 or 300 packs.
+As soon as there are 400 packs available, the torrent should be avaliable.
```
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: There already are 400 beatmap packs?

File: wiki/Beatmaps/en.md [hunk 28]
```diff
@@ -1,207 +1,72 @@
 # Beatmaps
 
-## What are "Beatmaps"?
+Beatmaps are game levels in [osu!](/wiki/osu!_glossary), like the stages in the [DS games](/wiki/Glossary/#ds-games). In their most basic form, a beatmap have a song and various [hit objects](/wiki/hit_objects) placed according to the rhythm of the music. They can also have a background image, video, and/or [storyboard](/wiki/storyboard).
 
-*Beatmaps listing page: <https://osu.ppy.sh/p/beatmaplist>*
+Beatmaps can be uploaded to the [BSS](/wiki/BSS) (Beatmap Submission System) and are then run through an approval process called [modding](/wiki/modding) in order to maintain the high standards of playability.
```
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: Uploaded by the BSS

File: wiki/Beatmaps/en.md [hunk 29]
```diff
@@ -1,207 +1,72 @@
...
+Both ranked and approved beatmaps requires approval from two [Beatmap Nominators](/wiki/Beatmap_Nominators). They can influence the stats on a player's profile which is what makes them different from a [pending](#pending) or [graveyarded](#graveyard) beatmap.
 
-#### Ranked
+### Ranked
 
-Ranked Beatmaps have a heart ![Heart icon](/wiki/shared/Heart.gif) icon on the thread and on the in-game song selection.
+Ranked beatmaps use the double upwards chevron icon (![Double upwards chevron icon](/wiki/shared/status/ranked.png)) in the song select screen.
 
-These beatmaps ideally fulfill all of the [ranking criteria](/wiki/Ranking_Criteria) to ensure that every player can expect a certain kind of standard when playing them.
+These beatmaps ideally fulfill all of the [ranking criteria](/wiki/ranking_criteria). This is to ensure that most players can expect a certain kind of standard while playing them.
```
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: Kind of standards?

File: wiki/Beatmaps/en.md [hunk 30]
```diff
@@ -1,207 +1,72 @@
...
 
-#### Qualified
+### Qualified
 
-Like *Pending* mixed with *Approved*, **Qualified** beatmaps are the cutting edge (of beatmaps), changing weekly, they could be the next, new and approved beatmap(s)! Notice: During this state, a beatmap will instantly return *Pending* if a mistake was found (and your scores will be wiped out). If they survive a week whilst in *Qualified*, they will become the next, new and approved beatmap(s)! (oh, sorry, did I say that already?)
+Qualified beatmaps use the check icon (![Check icon](/wiki/shared/status/qualified.png)) in the song select screen.
 
-#### Pending
+These beatmaps are the last stage of the beatmapping and modding process. Before the beatmap begins playing, a dialoge will display, noting the status of the beatmap will cause:
```
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: Are in the last stage

File: wiki/Beatmaps/en.md [hunk 31]
```diff
@@ -1,207 +1,72 @@
...
 
-Beatmaps packs, as the title suggests, are a .zip file containing beatmaps. The name of the pack is based on what they contain (for example, "Approved Beatmap Pack \#7" would only contain Approved Beatmaps). It is worth noting that the most packs are:
+- No [pp](/wiki/pp) will be awarded.
+- All scores will be deleted when the creator updates the beatmap.
 
--   Roughly either ~100MB (Taiko/osu!mania) or ~200MB (-/Approved).
--   Always contain videos (if the Beatmap has one).
--   Requires a program to open the .zip pack (like WinRAR, 7zip to name a few)
--   The beatmaps are either in the form of .zip or .osz file
+This status will give these beatmaps a leaderboard.
```
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: Give those beatmaps

File: wiki/Beatmaps/en.md [hunk 32]
```diff
@@ -1,207 +1,72 @@
...
-| Chart | Fortnightly Chart challenge songs |
-| Theme | General theme of the songs |
-| Artist/Album | Based on Artist/Album |
+_See also: [Beatmap Ranking Procedure](/wiki/Beatmap_Ranking_Procedure)_
 
-If you really want **to fully complete your song list** (not just downloading your favorite songs only), **download all the (-/Approved/Taiko/osu!mania) Beatmaps Packs** in the **Standard** category and nothing else. The (-/Approved/Taiko/osu!mania) Beatmaps Packs will contain all the songs in the other 3 categories.
+Pending beatmaps use the question mark icon (![Question mark icon](/wiki/shared/status/pending.png)) in the song select screen.
 
-The Beatmaps packs can be found and downloaded from:-
+A beatmap with a pending status is usually waiting to be [modded](/wiki/modded). [Creators](/wiki/Creators) must seek for [modders](/wiki/modders) from the community and at least two [Beatmap Nominators](/wiki/Beatmap_Nominators) to get their beatmapset out of this status.
```
Comment (TPGPL) [inline wiki/Beatmaps/en.md]: Creators into Mappers?

**Comment by TPGPL** [comment]
Nuke the translations?

File: wiki/Beatmaps/Packs/en.md (hunk 23 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: not a link, and idk 🤷‍♂️

File: wiki/Beatmaps/Packs/en.md (hunk 25 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: what's the link?

File: wiki/Beatmaps/Packs/en.md (hunk 26 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: but the whole section was deleted?

Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: but didn't you tell me to delete the section (along with its children)?

File: wiki/Beatmaps/Packs/en.md (hunk 25 above)
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: osu.ppy.sh/beatmaps/packs i think?

File: wiki/Beatmaps/Packs/en.md (hunk 26 above)
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: Oh, didnt see that those are its children; dont delete it then

File: wiki/Beatmaps/Packs/en.md (hunk 27 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/Packs/en.md]: 🤷‍♂️ I have no clue how the beatmap pack numbering works ?_?

File: wiki/Beatmaps/en.md (hunk 28 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/en.md]: by the BSS doesn't sound right :/

File: wiki/Beatmaps/en.md (hunk 29 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/en.md]:
```
expect a certain kind of standard
       ^
```
this signifies that it is singular?

File: wiki/Beatmaps/en.md (hunk 32 above)
Comment (MegaApplePi) [inline wiki/Beatmaps/en.md]: a gist of what ppy wrote, they are called "creators"; do not use "mappers" anymore

File: wiki/Beatmaps/Packs/en.md (hunk 27 above)
Comment (TPGPL) [inline wiki/Beatmaps/Packs/en.md]: These torrents contain the 10, 100, 200, 300, 400 or 500 packs.
As soon as there are 600 packs available, the torrent should be available. Probably?

**Pull request merged**